src/IMangrove.sol

```diff
@@ -12,29 +12,23 @@ import {MgvLib, MgvStructs, IMaker, OLKey} from "./MgvLib.sol";
 import "./MgvLib.sol" as MgvLibWrapper;
 
 interface IMangrove {
-  event Approval(
-    address indexed outbound_tkn,
-    address indexed inbound_tkn,
-    uint indexed tickScale,
-    address owner,
-    address spender,
-    uint value
-  );
+  event Approval(address indexed outbound_tkn, address indexed inbound_tkn, address owner, address spender, uint value);
```

Comment:
Should we use olKey here?

Reply:
Good question. Are approvals per pair or per OLKey? If it's per pair, then we probably shouldn't use olKey here

Reply:
they are per pair!

Reply:
> What is the OLKey created from? only outbound and inbound? or does it also have other information?

It also has the `tickScale`:

```Solidity
struct OLKey {
  address outbound;
  address inbound;
  uint tickScale;
}
```